src/contracts/UniversalDiscovery/Provider.php

```diff
@@ -64,7 +64,7 @@ public function getLocationGridViewData(
     public function getLocations(array $locationIds): array;
 
     /**
-     * @deprecated 4.6.0 The "\Ibexa\Contracts\AdminUi\UniversalDiscoveryProvider::getRestFormat()" method is deprecated, will be removed in 5.0.
+     * @deprecated 4.6.0 The "\Ibexa\Contracts\AdminUi\UniversalDiscovery\Provider::getRestFormat()" method is deprecated, will be removed in 5.0.
```

Comment:
Too much self reference. The following is enough.
```suggestion
     * @deprecated 4.6.0 Will be removed in 5.0.
```